idaes/power_generation/carbon_capture/mea_solvent_system/property_package/liquid_prop.py

```diff
@@ -0,0 +1,1282 @@
+
+    def _diffus(self):
+        """
+        Diffusivity of diffusing components
+        """
+        def rule_diffus(blk, i):
+            if i == 'CO2':
+              C_MEA = blk.conc_comp['MEA']*1e-3
+              return (blk._params.diffus_param[i,1] +
+                    blk._params.diffus_param[i,2]*C_MEA +
+                    blk._params.diffus_param[i,3]*C_MEA**2)*exp(
+                    (blk._params.diffus_param[i,4] +
+                     blk._params.diffus_param[i,5])/blk.temperature)
+            elif i == 'MEA':
+               C_MEA = blk.conc_comp['MEA']
+               return exp(blk._params.diffus_param[i,1] +
+                          blk._params.diffus_param[i,2]/blk.temperature +
+                          blk._params.diffus_param[i,3]*C_MEA)
+            elif i == "MEACOO-" or i == 'MEA+':
+               return exp(blk._params.diffus_param[i,1] +
+                          blk._params.diffus_param[i,2]/blk.temperature +
+                          blk._params.diffus_param[i,3]*log(blk.visc_d))
+
+        self.diffus = Expression(self._params.component_list_diffus,
+                                 rule=rule_diffus,
+                             doc='diffusivity of component i in liquid [m^2/s]')
+
+    def _k2_rxn(self):
+        '''
+        concentration based rate constant for overall reaction rate
+        Second order rate contant(Luo et al., 2015)
+        '''
+        def rule_k2_rxn(blk):
+            T = blk.temperature
+            C_MEA = blk.conc_comp['MEA']*1e-3
+            C_H2O = blk.conc_comp['H2O']*1e-3
+            return (2.003e10*exp(-4742/T)*C_MEA +
+                     4.147e6*exp(-3110/T)*C_H2O)*1e-3
+
+        self.k2_rxn = Expression(rule = rule_k2_rxn,
+                                doc="Second order rate contant [m3/mol.s]")
+
+    def _habs(self):
+        '''
+        Heat of absorption of CO2
+        '''
+        def rule_habs(blk):
+            if blk.config.parameters.config.process_type == 'Regenerator' or \
+               blk.config.parameters.config.process_type == 'Stripper':
+                return -97000
+            elif blk.config.parameters.config.process_type == 'Absorber':
+                return -84000
+        self.habs = Expression(rule = rule_habs,
+                                doc="CO2 heat of absorption [J/mol]")
+
+    def _hvap(self):
+        '''
+        Heat of vaporization of H2O
+        '''
+        def rule_hvap(blk):
+            c  = blk._params.heat_vaporization_param
+            T  = blk.temperature
+            Tc = blk._params.temperature_crit['H2O']
+            return 1000*c[1]*(1-T/Tc)**(c[2]+c[3]*T/Tc+c[4]*(T/Tc)**2)
+        self.hvap = Expression(rule = rule_hvap,
+                                doc="Heat of vaporization of H2O [J/mol]")
+
+    def _henry_N2O_analogy(self):
+        '''
+        Henry's constant N2O Analogy Jiru et.al (2012)
+        '''
+        def rule_henry_N2O_analogy(blk):
+            T      = blk.temperature
```

Comment:
Remove the excess spaces before the "=".

Reply:
Done. I am hoping that the PEP8 formatting will adjust all punctuations accordingly. so far the PEP checker now shows the spacing to be okay.